wurst/objects/abilities/mage/hypnotist/DreamEater.wurst

```diff
@@ -10,18 +10,17 @@ import SimError
 
 //@configurable constant int ABILITY_ID = 'A04Z' Old GUI
 @configurable constant int BUFF_ID = BUFF_HYPNOTIZED
-@configurable constant string EFFECT_PATH = Abilities.aIsoTarget
 
 let HP_RESTORED = 50.
 let MANA_RESTORED = 30.
 let MANA_EATEN = 50.
-let DAMAGE = 20.
+let DAMAGE = 40.
 let MANACOST = 10
 let COOLDOWN = 20.
 
 let ANIMATION_NAME = "channel"
 let TOOLTIP_NORM = "Dream Eater"
-let TOOLTIP_EXTENDED = ("Steals an enemy dream restoring {0} Health Point and {1} Mana. Also reduces enemy's Mana by {2} and deals {3} damage. "+
+let TOOLTIP_EXTENDED = ("Steals an enemy dream restoring {0} Health Point and {1} Mana. Also reduces enemy's Mana by {2} and deals {3} pure damage. "+
```

Comment:
This is a big change so I'm not going to be picky about individual things right now when they can be bulk updated later, but in the future, let's try to be consistent about wording about points. ITT usually uses "energy" instead of "mana" and because it doesn't say "energy points" ever, I wouldn't say "health points" either. In this case, I would say "restoring X health and Y energy".

Regarding the "pure damage", I think that's just because it was coded loosely. It should probably use actual damage instead of setting HP so that AMS comes into play.

Reply:
Writing down a proper norm may seem overkill, but it would help. I am dubious about the "energy" wording, new player don't immediately understand that energy == mana.